### DISCUSSION TITLE
How to customize the arrow style

### DISCUSSION THREAD

**Discussion by 19Qingfeng** [gilbarbara/react-joyride#747] [Q&A] [answered]
When I want to customize the style of the arrow, is there any configuration item exposed.

I looked in the documentation and found only the color configuration.

I'm looking forward for your help, thank your.

**Comment by eugenegodun** [comment, marked as answer]
https://docs.react-joyride.com/#create-awesome-tours-for-your-app
>It uses [react-floater](https://github.com/gilbarbara/react-floater) for positioning and styling.

Just check [react-floater](https://github.com/gilbarbara/react-floater) styling props and then use `floaterProps` for styling `react-floater` inside `react-joyride`
